tmux/README.md

```diff
@@ -42,3 +42,16 @@ The next keypresses you make will be interpreted as a command to `tmux`, which a
 - [source and documentation](https://github.com/tmux/tmux)
 - [FAQ](https://github.com/tmux/tmux/wiki/FAQ)
 - [man page](http://man.openbsd.org/OpenBSD-current/man1/tmux.1)
+
+## Some tmux shortcuts
+- Open a new session
+    - tmux new
+    - tmux new dev -> named session called "dev"
+- Tmux command mode (Ctrl + b)
```

Comment:
Doesn't this conflict with the actual config that redefines the control code?

Reply:
Could you clarify this point please. I have so far had no issues with named sessions.

Reply:
The tmux config that is in the tree uses a different control sequence, not `cntl + b`, so including documentation that implies it is `cntl + b` is not quite right.

Reply:
In the tmux config I use `cntl+a` is the secondary `<prefix>` but `cntl+b` still works.

Reply:
I changed it to `cntl + t`. I had assumed, erroneously, that since unbind was not used `cntl + b` would still work.

Reply:
Commit e288d5c